pkg/tool/gen-disk-skus-map.go

```diff
@@ -45,53 +47,58 @@ var (
 	VCPUsCapability                      = "vcpus"
 )
 
+var (
+	outputFile  = flag.String("filepath", "../azureutils/azure_dd_max_disk_count.go", "path to the file to apply the update.")
+	packageName = flag.String("package-name", "azureutils", "flag to indicate the package name for the file being updated.")
+	generateAll = flag.Bool("generate-all", false, "generate both disk and sku maps. Otherwise only generates node sku name to disk count map.")
+)
+
 func init() {
 	klog.InitFlags(nil)
+	flag.Parse()
 }
 
 func main() {
 
 	boilerPlate := `/*
 Copyright 2021 The Kubernetes Authors.
```

Comment:
Adding the "generated" comment below causes the `./hack/verify-boilerplate.sh` script to compare the generated file against the `boilerplate.generatego` file which omits the copyright year. You'll need to remove that to get the verify tests to pass.

Reply:
Yep, although removing the DO NOT EDIT was enough.